src/rust/engine/fs/store/src/remote.rs

```diff
@@ -28,8 +28,8 @@ use workunit_store::{in_workunit, Metric, ObservationMetric, WorkunitMetadata};
 pub struct ByteStore {
   instance_name: Option<String>,
   chunk_size_bytes: usize,
-  upload_timeout: Duration,
-  rpc_attempts: usize,
+  _upload_timeout: Duration,
+  _rpc_attempts: usize,
```

Comment:
Some of these probably justify TODOs, since they can't possibly have `Drop` impls... although as you say: the `Runtime` does.